### ISSUE TITLE
Create guidelines for volunteer staffers to start having access to resources and assets such as email, zoom, etc.

### ISSUE THREAD

**Issue by rcscastillo** [ProgressiveCoders/functions#202]
#### Description

Build guidelines and necessary process structure for volunteers to gain access. This proposal aims spark a conversation within the volunteer staff team on how best to proceed gaining access for specific assets and resources.  This is an iteration of #193.

The aim is to provide a clear accountability and transparency structure to providing resources to our community members. The first step could be a request for a progcode email. Then it could be a per-tiered approach (eg. volstaff interested in community will have access to calendly (one-on-one), and other resource sets), comms team would get access to hootsuite/actionsprout/CRMs etc, and partnerships could get access to calendly as well. 

Another level could be done when accessing private information about members (eg. member vetting) which would need added consent to the once currently holding that information. 

#### Problem

Right now, there is no clear guideline on how to provide assets and resources to empower volunteer staffers. We need to have a clear, transparent, and accountable process that will enable people who want to contribute become better members.

#### Benefit

This will provide more accountability and transparency in the process, safeguard private information with the high level of trust, and expand our volunteer staffer base to the targeted 15 members.

#### Plan

- [ ] Consent to continue
- [ ] Schedule discussion
- [ ] Collaboratively build the guideline from the discussion
- [ ] Start another issue for the implementation of the guideline/

#### Decision Making

On this stage, `consent-to-continue`

#### Reference link(s)

...

**Comment by dave-mahler** [comment]
Could we start by compiling a list of which staffers have access to which resources, and what they use them for? Good way for those of us without access to get a high level overview

**Comment by rcscastillo** [comment]
Sounds good @dave-mahler

**Comment by jpb5013** [comment]
This is great. I think we should also more explicitly define requirements for people to become volunteer staffers. Creating a Wiki based on my list in the issue you referenced could be a starting point.

**Comment by dave-mahler** [comment]
I think we're better off not thinking of it as requirements to become a staffer. Rather, is about the work that we need help with and offering guidance to those interested in helping out

**Comment by jpb5013** [comment]
Good point, requirements is probably poor choice of work. Like anything
else, it's the process, we need to have something concrete. This doesn't
mean people shouldn't be able to contribute as soon as they're a community
member, however for dedicated resources and access to potentially sensitive
information I think it's reasonable to have extended expectations.

**Comment by tktino** [comment]
I like to request training and tools on how to assist and engage with partners/projects in respect to their tac status.
